sessionctx/variable/sysvar.go

```diff
@@ -391,7 +391,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, "binlog_direct_non_transactional_updates", "OFF"},
 	{ScopeGlobal, "innodb_change_buffering", "all"},
 	{ScopeGlobal | ScopeSession, "sql_big_selects", "ON"},
-	{ScopeGlobal | ScopeSession, CharacterSetResults, "latin1"},
```

Comment:
We are planning to change it to "utf8". Will this solve your problem?

Reply:
#7198 is okay, I will try to use @coocood 's suggestion.